src/MessageDialog.Shared/MessageDialog.Shared.projitems

```diff
@@ -6,7 +6,7 @@
     <SharedGUID>3ab437c2-4f5b-4e7c-ad5c-563e03fb9a4a</SharedGUID>
   </PropertyGroup>
   <PropertyGroup Label="Configuration">
-    <Import_RootNamespace>Shared</Import_RootNamespace>
+    <Import_RootNamespace>MessageDialog.Shared</Import_RootNamespace>
```

Comment:
Actually, the root namespace should be `MessageDialog`, not `MessageDialog.Shared`.

Reply:
true!